process/block/baseProcess.go

```diff
@@ -128,6 +133,108 @@ type bootStorerDataArgs struct {
 	epochStartTriggerConfigKey []byte
 }
 
+func NewBaseProcessor(arguments ArgBaseProcessor) (*baseProcessor, error) {
```

Comment:
does this needs to be exported? if yes, add missing comment

Reply:
done